inference-engine/src/mkldnn_plugin/nodes/mkldnn_permute_node.cpp

```diff
@@ -148,6 +148,13 @@ void MKLDNNPermuteNode::getSupportedDescriptors() {
     std::vector<int> layerOrder = layer->GetParamAsInts("order");
     for (auto ord : layerOrder)
         order.push_back(static_cast<size_t>(ord));
+
+    if (order.empty()) {
```

Comment:
You fixed the CPU plugin. But what about others?

Reply:
At the current we have such tests only for CPU.
I think if these tests will be extended for other plugins, plugin owners will fix implementations.